specs/deneb/p2p-interface.md

```diff
@@ -300,6 +303,11 @@ Clients MUST support requesting sidecars since `minimum_request_epoch`, where `m
 Clients MUST respond with at least one sidecar, if they have it.
 Clients MAY limit the number of blocks and sidecars in the response.
 
+Clients SHOULD include a sidecar in the response as soon as it passes the gossip validation rules.
+Clients MUST NOT respond with sidecars that fails gossip validation rules.
+Clients MUST NOT respond with sidecars related to blocks that fail gossip validation rules.
```

Comment:
```suggestion
Clients SHOULD NOT respond with sidecars related to blocks that fail gossip validation rules.
```
I believe we can’t use MUST NOT because a block isn’t always around, with MUST NOT a client won’t be able to respond with a sidecar until it pulls a block and runs gossip validation checks on it

Reply:
ah, I see